contrib/hawq-docker/Makefile

```diff
@@ -227,3 +229,85 @@ distclean:
 		docker network rm $(NETWORK) 2>&1 >/dev/null || true; \
 	fi
 	@echo "Distclean Done!"
+
+hawq:
+	@echo "Logging into ${CLUSTER_ID}-namenode container"
+	@if [ ! -z "`docker ps -a --filter="name=${CLUSTER_ID}-namenode" | grep -v CONTAINER`" ]; then \
+		docker exec \
+			-e "HAWQ_HOME=$(HAWQ_HOME)" \
+			-e "JAVA_TOOL_OPTIONS=$(JAVA_TOOL_OPTIONS)" \
+			-u gpadmin --privileged -it ${CLUSTER_ID}-namenode "make-hawq.sh"; \
+	else \
+		echo "${CLUSTER_ID}-namenode container does not exist!"; \
+	fi
+
+start-hawq:
```

Comment:
It's better to add make dependencies for this option, or add description what's the make sequence to call this option.

Reply:
Fixed